content/docs/command-reference/fetch.md

```diff
@@ -161,10 +161,10 @@ $ dvc fetch
 
 $ tree .dvc/cache
 .dvc/cache
-├── 38
-│   └── 63d0e317dee0a55c4e59d2ec0eef33
-├── 42
-│   └── c7025fc0edeb174069280d17add2d4.dir
+├── 20
+│   └── b786b6e6f80e2b3fcf17827ad18597.dir
+├── 32
+│   └── b715ef0d71ff4c9e61f55b09c15e75
```

Comment:
OK but make sure all the mentions of these values match. There are some in line 182, for example.

Reply:
`32b71..` is not related to any file present in workspace, not there in `dvc.lock` either. So I have kept only `20b7...` in line 182.

Reply:
Hmm so why is `32b715e in the cache above? It's supposed to be the `model.pkl` file. Please don't just remove the other file from line 182, both should be mentioned to explain the cache tree above. Just update the correct hash values...